### PR TITLE
added 2grid system

### PR DESCRIPTION
**Pull request by Polleke007**

### CODE REVIEW

File: src/mapping/sanity/expert-advice.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Button } from '@src/components/features/decoupled-components/button/button';
```
Comment (ceneva) [inline src/mapping/sanity/expert-advice.tsx:1]: In the mapping folder shouldn't be any component code, all data/query related code should go in folder data, all component code should be in folder component, the only thing that the mapping is supposed to do is to connect the sanity specific data to the generic component data it belongs to, it transfers (or slightly alters) the data wherever needed. Can you please split the code into the right parts? Thanks. Feel free to ask questions if needed ;-)

File: src/mapping/sanity/featured-article.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import {PortableText} from '@portabletext/react'
```
Comment (ceneva) [inline src/mapping/sanity/featured-article.tsx:1]: Same comment as for expert advice

File: src/mapping/sanity/featured-article.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+interface FeaturedProps {
+  subtitleRaw: any[];
+  title: string;
+  buttonLink: string;
+  heading: string;
+}
+
+export function FeaturedArticle(props: any) {
+  console.log("props", props);
```
Comment (ceneva) [inline src/mapping/sanity/featured-article.tsx:11]: Please remove console logs

File: src/mapping/sanity/two-grid.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (ceneva) [inline src/mapping/sanity/two-grid.tsx:1]: Same comment as for expert advice

**Review by ceneva — CHANGES_REQUESTED** [review]

**Review by DavithkbY — CHANGES_REQUESTED** [review]
Fix Celien's comments

**Pull request merged**